factory.md

```diff
@@ -0,0 +1,43 @@
+# Factory
+Build pools with factory contract. The purpose of this contract is to enable listing pools in a fully decentralized way. Factory ensures the uniqueness of pools with given configuration therefore less possibilities for liquidity defragmentation.
+
+# Constant Product Factory
+
+## Description of global state
+
+| Constant name         | Description                                                                                |
+| --------------------- | ------------------------------------------------------------------------------------------ |
+| ADMIN                 | Admin address to be set in built pools                                                     |
+| TREASURY              | Treasury address to be set in built pools                                                  |
+| POOL_CONTRACT_VERSION | Version of the pool contract built by factory                                              |
+| ALLOWED_FEE_BPS       | Fee bps allowed to be set in built pools                                                   |
+| ALLOWED_PACT_FEE_BPS  | 1:1 mapping with ALLOWED_FEE_BPS of pact fee bps to be set in pool when fee bps is chosen  |
+
+## Pools' uniqeuness
+Pool's uniqeuness is determined by checking the existence of a box. The box's name is created by concatenating: primary asset id, secondary asset id, fee bps and version of pool to be built. 
+The box contains the app id of a pool with a given configuration.
+
+## List of possible transaction groups
+The available method signatures from the ABI are described in [../factory_interface.json](./factory_interface.json).
```

Comment:
Add following information: `ARC-4 specification is available under the "contract" key in the JSON file`